src/BridgedRoom.ts

```diff
@@ -292,22 +292,30 @@ export class BridgedRoom {
         this.addRecentSlackMessage(`reactadd:${relatesTo.key}:${id}:${event.slackTs}`);
 
         // TODO: This only works once from matrix if we are sending the event as the
-        // bot user.
+        // bot user. Search for #fix_reactions_as_bot.
```

Comment:
What does `Search for #fix_reactions_as_bot` mean?

Reply:
I added this to several line to easier find them when the feature gets implemented.
As I expect this to be a feature we'll want to implement, I wanted to add these bookmarks.

Reply:
Removed